xonsh/main.py

```diff
@@ -150,6 +150,10 @@ def premain(argv=None):
         setproctitle(' '.join(['xonsh'] + sys.argv[1:]))
     builtins.__xonsh_ctx__ = {}
     args, other = parser.parse_known_args(argv)
+    if not args.args and other and other[0].startswith('-'):
+        print('xonsh: error: invalid argument {!r}'.format(other[0]))
```

Comment:
there are two more `print` in main that print errors, should i change them too? @scopatz